en/identity-server/5.10.0/docs/setup/deployment-guide.md

```diff
@@ -251,7 +251,104 @@ WSO2 supports the following membership schemes for clustering
+
+            When WSO2 products are deployed in clustered mode on Amazon EC2 instances, it is recommended to use the AWS clustering mode. Open the `deployment.toml` file (stored in the `<IS_HOME>/repository/conf/` directory) and 
+            do the following changes.
+
+            1. Apply the following configuration parameters and update the values for the server to enable AWS 
+            clustering.
+                    ```toml
+                    [clustering]
+                    membership_scheme = "aws"
+                    domain = "wso2.carbon.domain"
+                    local_member_host = "10.0.21.80"
+                    local_member_port = "5701"
+                    ```
+                The port used for communicating cluster messages has to be any port number between 5701 and 5800. The 
+                local member host must be set to the IP address bound to the network interface used for communicating 
+                with other members in the group (private IP address of EC2 instance).
+
+            2. Apply the following parameters to update the values to configure clustering properties.
+                    ```toml
+                    [clustering.properties]
+                    accessKey = "***"
+                    secretKey = "***"
+                    securityGroup = "security_group_name"
+                    region = "us-east-1"
+                    tagKey = "a_tag_key"
+                    tagValue = "a_tag_value"  
+                    ```
+                It's recommended to add all the nodes to the same security group. The AWS credentials and security 
+                group depend on your configurations in the Amazon EC2 instance. The `tagKey` and `tagValue` are 
+                optional and the rest of the above parameters are mandatory. 
+
+            3. To provide specific permissions for creating an access key and secret key for only this AWS clustering attempt, use the custom policy block given below.
+                See the [AWS documentation](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_managed-policies.html) for details on how to add the custom IAM policy. 
+                    Attach this to the user account that will operate AWS clustering in your WSO2 IS. The access key and secret key can only be used to list EC2 instance details in the AWS account.
+                    ```json
+                    { "Version": "2012-10-17",
+                    "Statement":
+                    [
+                    {
+                        "Effect": "Allow",
+                        "Action":
+                            [
+                            "ec2:DescribeAvailabilityZones",
+                            "ec2:DescribeInstances"
+                            ],
+                            "Resource": [ "*" ]
+                    }
+                    ]
+                    }
+                    ```
+
+        ??? tip "Click to see the instructions for Kubernetes membership scheme"
```

Comment:
These instructions are for updated pack right? For anyone using a vanila pack, these instructions won't be valid. Shall we update this similar to [5.11 documentation](https://is.docs.wso2.com/en/5.11.0/setup/deployment-guide/#:~:text=When%20IS%20nodes%20are%20deployed%20in%20a%20clustered)?

Reply:
Added